src/appengine/handlers/testcase_detail/find_similar_issues.py

```diff
@@ -23,47 +22,46 @@ class Handler(base_handler.Handler):
   """Handler that finds similar issues."""
 
   @staticmethod
-  def get_issues(testcase, filter_type):
+  def get_issues(issue_tracker, testcase, only_open):
     """Get similar issues. It is used by self.process() and
     handler.testcase_detail.FindSimilarIssuesHandler.get()"""
-    itm = helpers.get_issue_tracker_manager(testcase)
-
     issues = issue_tracker_utils.get_similar_issues(
-        testcase, can=filter_type, issue_tracker_manager=itm)
+        issue_tracker, testcase, only_open=only_open)
 
     items = []
     for entry in issues:
       items.append({
-          'owner': entry.owner,
+          'owner': entry.assignee,
           'reporter': entry.reporter,
-          'security': entry.has_label_containing('security'),
           'status': entry.status,
-          'summary': entry.summary,
-          'updated': utils.time_difference_string(entry.updated),
+          'title': entry.title,
           'id': entry.id
       })
 
     items = sorted(items, key=lambda k: k['id'])
-    issue_url = issue_tracker_utils.get_issue_url(testcase)
-    return items, issue_url
+    return [{
+        'issue': item,
+        'url': issue_tracker.issue_url(item),
+    } for item in items]
 
   @handler.get(handler.JSON)
   @handler.check_admin_access_if_oss_fuzz
   @handler.check_testcase_access
   def get(self, testcase):
     """Find similar issues."""
     filter_type = self.request.get('filterType')
+    only_open = filter_type == 'open'
```

Comment:
nit: should filters be defined as enum in issue_management, then we can just say like filter_type == issue_filters.OPEN_BUGS

Reply:
This string representation for filter_type is specific to this handler and the frontend, so I'm not sure if it makes sense to add it to issue_management.